src/xunit.performance.api/PerformanceTestConfig.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+
+
+namespace Microsoft.Xunit.Performance.Api
+{
+    public class PerformanceTestConfig
+    {
+        public PerformanceTestConfig() 
+        {
+            Iterations = 30;
```

Comment:
Great question.  I'm torn here.  On one hand, I think default values are good because otherwise people have to decide what they should be.  On the other hand, I'm not sure how we pick good defaults.

My suspicion is that we want to have defaults because we can change them if we discover that they're not good.  However, if we force values through the constructor then we can't change it unless we go change everyone's code.  I think that this is less interesting for timeout.  People can make an educated guess on that.  But iterations is much more contentious and unclear.  Maybe we require timeout and have a default value for iterations?